recipe/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   sha256: 1524b840e42cf3b909e8f8df67c1724012c7dc7f9d076d4feef2d3eff031e8a0
   patches:
       - omit-local-site-packages.patch  # [osx]
+      - osx64-dist.patch                # [osx and x86_64]
```

Comment:
Though I don't think OS X 32-bit has been supported for a bit. I don't have any problem with adding `x86_64`.